public/js/songs.js

```diff
@@ -0,0 +1,63 @@
+const songsList = document.getElementsByClassName('catalog-grid')[0];
+const searchBar = document.getElementById('searchBar');
+let songs = [];
+
+console.log(searchBar);
+
+// After an user type a character in the searchbox
+searchBar.addEventListener('keyup', (e) => {
+  const searchString = e.target.value.toLowerCase();
+
+  const filteredSongs = songs.filter( song => {
+    console.log(song.title);
+    console.log(song.artist);
+
+    return (
+      song.title.toLowerCase().includes(searchString) || 
+      song.artist.toLowerCase().includes(searchString) ||
+      song.album.toLowerCase().includes(searchString)
```

Comment:
⛳ - dynamically check every key for the search string and if you refactor or add new fields it won't need to be updated.
```suggestion
return Object.values(song).some(value => typeof value === 'string' ? value.toLowerCase().includes(searchString) : false)
```